tools/cli/src/args.rs

```diff
@@ -50,17 +50,29 @@ pub enum Command {
 
 #[derive(Clone, Args)]
 pub struct TaskQueueArgs {
+    /// Task queue address. If not provided, then it will be read
+    /// from the environment variable LOCAL_TASK_QUEUE_ADDRESS or TEST_TASK_QUEUE_ADDRESS
+    /// depending on the target environment
+    #[clap(long)]
+    pub address: Option<String>,
+
     #[command(subcommand)]
-    command: TaskQueueCommand,
+    pub command: TaskQueueCommand,
 }
 
 #[derive(Clone, Subcommand)]
 pub enum TaskQueueCommand {
     /// Commands for task queue
     AddTask {
-        // set the default
+        /// The body of the task, must be valid JSON
+        #[clap(short, long)]
+        body: String,
+        /// Human-readable description of the task
```

Comment:
Is this required by the contract? I think we can make it optional.

Reply:
yeah, right now it is: https://github.com/Lay3rLabs/avs-toolkit/blob/0c09a3b7aa9ecb3d5db80368ee3693173b127179/packages/apis/src/tasks.rs#L67

want me to make that change to the contract as part of this PR?